src/main/java/seedu/duke/template/LcTemplate.java

```diff
@@ -112,6 +113,7 @@ public String toString() {
      */
     @Override
     public LoadComponent getComponent(String component) throws DukeException {
+        assert component.equals("c") || component.equals("r");
```

Comment:
The test for this method in the RcTemplate_Test will not run because it tries to catch DukeException but the assert keyword throws a different kind of execption.

Reply:
Might consider removing this, since it is already tested by JUnit?

Reply:
Right, will remove and push again. Thanks for pointing out!